README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
I much prefer `npm install rxjs`. Works great for TypeScript. I guess this is a naive question, but why wouldn't we prefer the shorter version?